server/route.go

```diff
@@ -710,6 +983,9 @@ func (s *Server) createRoute(conn net.Conn, rURL *url.URL) *client {
 	// Initialize
 	c.initClient()
 
+	// Remember numAcccounts we are sending to the other side since things can change
+	// and we want to be deterministic about who sends first account list.
+
```

Comment:
Comment no longer relevant or forgot to add code :-)

Reply:
yep, removed..